src/main/java/fi/aalto/cs/apluscourses/intellij/model/APlusProject.java

```diff
@@ -30,6 +32,12 @@ public ModuleManager getModuleManager() {
     return ModuleManager.getInstance(project);
   }
 
+  @Nullable
+  public ModuleRootManager getModuleRootManager(@NotNull String moduleName) {
```

Comment:
could this have a JavaDoc, please :wink:

Reply:
I'll add that.